molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/ElasticsearchRepositoryCollection.java

```diff
@@ -92,11 +92,12 @@ public void deleteEntityMeta(String entityName)
 	@Override
 	public void addAttribute(String entityName, AttributeMetaData attribute)
 	{
-		EntityMetaData entityMetaData = dataService.getEntityMetaData(entityName);
+		EntityMetaData entityMetaData = dataService.getMeta().getEntityMetaData(entityName);
```

Comment:
See 4 lines below, but I'll rewrite that bit anyhow, was planning to but forgot.
The change here is probably unnecessary, a artefact of a attempt to understand/fix what was going on.